Dockerfile.gpu

```diff
@@ -0,0 +1,17 @@
+# Docker image for running examples in Tensorflow models.
+# base_image depends on whether we are running on GPUs or non-GPUs
+FROM nvidia/cuda:9.0-cudnn7-runtime-ubuntu16.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    ca-certificates \
+    build-essential \
+    git \
+    python \
+    python-pip \
+    python-setuptools
+    
+RUN pip install tensorflow-gpu
```

Comment:
The models now run off of the nightly builds-- tf-nightly and tf-nightly-gpu.

Reply:
It will be useful to pin specific releases of models repo to tf releases. I'll update this PR to point to nightly.